test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -85,7 +85,7 @@ const (
 var (
 	defaultMetricsCollectorConfig = metricsCollectorConfig{
 		Metrics: map[string]*labelValues{
-			"scheduler_framework_extension_point_duration_seconds": {
```

Comment:
This is not right, the metric should match the one exported by the scheduler [1], otherwise we will not be able to look it up.

[1] https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/metrics/metrics.go#L149

Reply:
@ahg-g Then, what should I do, in the issue it is mentioned to update the naming convention to avoid confusion.